extensions/workouts/src/api/client.ts

```diff
@@ -194,3 +196,43 @@ export const getClubActivities = async (clubId: string, page = 1, pageSize = PAG
     throw new Error(error);
   }
 };
+
+export const createActivity = async (activityValues: StravaManualActivity) => {
+  console.log("activityValues");
```

Comment:
remove logs

Reply:
I agree on the return action. I spent some time on trying to clear the form values, but could not find a nice way to get I working, so I thought I would leave it our for now.

Reply:
you could probably make the form controlled to be able to reset the state, but don't think it blocks getting this merged